src/Apps/ViewingRoom/Components/ViewingRoomTabBar.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { Flex, color, Color, FontWeights, Box, Sans } from "@artsy/palette"
+import { useIsRouteActive } from "Artsy/Router/useRouter"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import { LinkProps } from "found"
+import styled from "styled-components"
+
+interface ViewingRoomTabBarProps {}
+
+export const ViewingRoomTabBar: React.FC<ViewingRoomTabBarProps> = props => {
+  return (
+    <Flex width="100%" justifyContent="center" id="tabBarAnchor">
+      <Flex width="66%" height={50}>
+        {/* FIXME: Update with `:id` prop when ready */}
+        <Tab to={`/viewing-room`}>Statement</Tab>
+        <Tab to={`/viewing-room/works`}>Works</Tab>
+      </Flex>
+    </Flex>
+  )
+}
+
+const Tab: React.FC<LinkProps> = ({ children, to }) => {
+  const active = useIsRouteActive(to)
+
+  const borderBottom = active
+    ? `2px solid ${color("black100")}`
+    : `1px solid ${color("black10")}`
```

Comment:
Does the change in border width cause the content to shift up a bit?

Reply:
Nope, not that I can see: https://staging.artsy.net/viewing-room